### DISCUSSION TITLE
Idiomatic way to parse multiple variables when passed through stdin.

### DISCUSSION THREAD

**Discussion by st0le** [pomdtr/sunbeam#80] [Q&A] [answered]
I'm writing an extension and I need to pass 2 variables (potentially more) for a type: run action.

I'm by no means a bash/sh guru. What would be the best way to parse multiple values from .params into variables that I can pass into a command.

Of course, I can only read stdin once.

**Comment by b-per** [comment, marked as answer]
@pomdtr and I discussed it in [this issue](https://github.com/pomdtr/sunbeam/issues/72)

The idea is to load all of them in a bash variable and then query this variable
